ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OAlgorithm.scala

```diff
@@ -89,8 +89,6 @@ abstract class H2OAlgorithm[P <: Model.Parameters : ClassTag]
     val model = H2OMOJOModel.createFromMojo(mojoData, Identifiable.randomUID(binaryModel._parms.algoName()))
 
     // pass some parameters set on algo to model
-    model.setFeaturesCols(binaryModel.modelDescriptor().features())
-    model.setLabelCol(getLabelCol())
```

Comment:
As we discussed, we won't put label col into the final model. Also we don't do it for mojo pipeline & automl & gridsearch

Reply:
Ditto